code/modules/organs/organ_internal.dm

```diff
@@ -130,7 +130,25 @@
 
 	if(owner.life_tick % PROCESS_ACCURACY == 0)
 
-		//High toxins levels are dangerous
+		//A liver's duty is to get rid of toxins
+		if(owner.getToxLoss() > 0 && owner.getToxLoss() <= 3)
+			owner.adjustToxLoss(-0.2) //there isn't a lot of toxin damage, so we're going to be chill and slowly filter it out
+		else if(owner.getToxLoss() > 3)
+			if(is_bruised())
+				//damaged liver works less efficiently
+				owner.adjustToxLoss(-0.5)
+				if(!owner.reagents.has_reagent("anti_toxin")) //no damage to liver if anti-toxin is present
+					src.damage += 0.1 * PROCESS_ACCURACY
+			else if(is_broken())
+				//non-functioning liver doesn't fix toxins
```

Comment:
empty else statement?

Reply:
Would it be better to have it deal toxin damage, instead of doing nothing?

Reply:
i would say so tbh

Reply:
commit die